lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, Type
+
+from lisa import schema
+from lisa.environment import Environment
+from lisa.util import InitializableMixin, subclasses
+from lisa.util.logger import get_logger
+from lisa.util.subclasses import Factory
+
+from ..build import Build
+from ..schema import ClusterSchema
+
+
+class Cluster(subclasses.BaseClassWithRunbookMixin, InitializableMixin):
+    def __init__(
+        self,
+        runbook: ClusterSchema,
+    ) -> None:
+        super().__init__(runbook=runbook)
+        self.cluster_runbook: ClusterSchema = self.runbook
+        self._log = get_logger("cluster", self.__class__.__name__)
+
+    def copy(self, source_path: str) -> None:
```

Comment:
The copy method should be the same for all clusters. And as the logic, it should be an action of platform, not the cluster.

Reply:
updated